src/Models/BaseElement.php

```diff
@@ -345,7 +349,7 @@ public function getCMSFields()
 
                 $fields->insertBefore($styleDropdown, 'ExtraClass');
 
-                $styleDropdown->setEmptyString(_t(__CLASS__.'.CUSTOM_STYLES', 'Select a style..'));
+                $styleDropdown->setEmptyString(_t(__CLASS__.'.CUSTOM_STYLES', 'Select a style...'));
```

Comment:
```suggestion
                $styleDropdown->setEmptyString(_t(__CLASS__.'.CUSTOM_STYLES', 'Select a style..'));
```

Reply:
Updated the default string won't match the yml - https://github.com/silverstripe/silverstripe-elemental/blob/4/lang/en.yml#L11 - unfortunately changing strings in en.yml is pretty fraught and can cause the corresponding translations in transifex are likely revert, so we'd need to create an entirely new string here.  For this reason we should just keep this as is.

Reply:
fair enough, what is the easiest way to revert this?

Reply:
I have fixed this properly - hopefully I did it right.

Reply:
Adding new translations really needs to happen via transifex, not updating the yml directly (it's not an ideal process).

The existing translations are a mixture of '..' and '...', rather than updating yml, I think the following would make more sense (untested):
```
$str = _t(__CLASS__.'.CUSTOM_STYLES', 'Select a style..');
$str = preg_replace('#([^\.])\.\.$#', '$1...', $tr);
$styleDropdown->setEmptyString($str);
```

Reply:
As noble as fixing the typo in the default string (and the translations) is, I think it shouldn't be a part of this PR.
Ideally, a PR addresses one main thing only.

Reply:
I agree, but I am short on time so I will leave it up to you to fix then. The double dot looks like someone did not finish something (maybe because they ran out of time)?